### PR TITLE
Upgrade to 0 14 version

### PR DESCRIPTION
**Pull request by isaacmg410**
Related with #31 
- [x] Upgrade version Decidim to 0.14.4
- [x] add missing locales
- [x] rails decidim:upgrade to add missing migrations
- [x] update the overwritten files
- [x] fix documentation of overwritten files

### CODE REVIEW

**Comment by isaacmg410** [comment]
@tramuntanal when you can, please, make a review of this upgrade

File: decidim-home/lib/decidim/home.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 require "decidim/home/engine"
-
```
Comment (tramuntanal) [inline decidim-home/lib/decidim/home.rb]: I find many changes of this kind (removal of empty lines). Do you do them on purpose? If not, better keep the code as is if we're not improving readability.

File: decidim-process-extended/decidim-process-extended.gemspec [hunk 2]
```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.2", "< 6.0.x"
+  s.add_dependency "decidim-core"
 
   s.add_development_dependency "sqlite3"
```
Comment (tramuntanal) [inline decidim-process-extended/decidim-process-extended.gemspec]: we should remove this dependencies to sqlite3, as Decidim depends directly from PostgreSQL.

File: decidim-process-extended/decidim-process-extended.gemspec [hunk 3]
```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.2", "< 6.0.x"
+  s.add_dependency "decidim-core"
```
Comment (tramuntanal) [inline decidim-process-extended/decidim-process-extended.gemspec]: Why this new dependency?

File: decidim-regulations/app/views/decidim/participatory_processes/participatory_processes/_order_by_processes.html.erb [hunk 4]
```diff
@@ -1,19 +1,19 @@
 <% is_process = request.original_fullpath.include? "/processes" %>
 
 <% if is_process %>
-
-  <div class="row collapse order-by">
+  <%= cell "decidim/participatory_processes/process_filters", process_count_by_filter, current_filter: filter %>
+  <!-- <div class="row collapse order-by">
```
Comment (tramuntanal) [inline decidim-regulations/app/views/decidim/participatory_processes/participatory_processes/_order_by_processes.html.erb]: Please remove useless code.

File: decidim-process-extended/decidim-process-extended.gemspec (hunk 3 above)
Comment (isaacmg410) [reply decidim-process-extended/decidim-process-extended.gemspec]: for some new functionalities in decidim-core

**Comment by isaacmg410** [comment]
@tramuntanal requested changes was applied! Can you review it again please?

**Pull request merged**